foundry.toml

```diff
@@ -1,3 +1,8 @@
+[fmt]
+line_length = 120
+number_underscore = 'thousands'
+wrap_comments = false
```

Comment:
@bagelface is this in line with your formatting preferences?

Reply:
tbh I dont really have strong opinions about formatting so long as there is consistency